lib/Sema/TypeCheckEffects.cpp

```diff
@@ -116,18 +130,34 @@ class AbstractFunction {
     return TheExpr;
   }
 
+  ConcreteDeclRef getDeclRef() {
+    return declRef;
+  }
+
+  ModuleDecl *getModuleContext() {
+    assert(getKind() == Kind::Function);
+    return TheFunction->getModuleContext();
```

Comment:
Let's always use the DC that we're type checking to get the module instead

Reply:
hmm that isn't used!, deleting it